.travis.yml

```
@@ -116,3 +142,9 @@ jobs:
       script:
         - make image/build/scorecard-proxy
         - make image/push/scorecard-proxy
+
+    # Build and deploy helm multi-arch manifest list
+    - <<: *before-deploy-manifest
```

This should match the name of the YAML anchor (I admit this part is super confusing) :)

```suggestion
    - <<: *manifest-deploy
```

---

yah, that was a typo on my part. i'd meant to make them match. couldn't figure out why my internal travis jobs weren't running and that was it!